Test/Utils/Testenv/Testbed.Impl.ps1

```diff
@@ -13,6 +13,10 @@ class Testbed {
     [String] $VHostName
     [PSSessionT] $Session = $null
 
+    # Keep in mind that this vairiable will only be initialized when data adapter exists at the beginning of run
```

Comment:
I believe we don't need this comment anymore

Reply:
Thanks